web/src/components/organisms/Workspace/Settings/index.tsx

```diff
@@ -0,0 +1,17 @@
+import WorkspaceSettingsMolecule from "@reearth-cms/components/molecules/WorkspaceSettings";
+
+import useHooks from "./hooks";
+
+const WorkspaceSettings: React.FC = () => {
+  const { handleWorkspaceUpdate, handleWorkspaceDelete, workspaceName } = useHooks();
```

Comment:
I know its small, but please try to keep imports, props, etc in the same order as their type or passed order. In this case, workspaceName is returned first in the hooks file, so should be first here.
Additionally, if you can keep functions AFTER variables, that would be perfect (just like here will be once workspaceName is moved)

Reply:
@KaWaite , I'll keep this in mind.